docs/api/api.md

```diff
@@ -17,6 +17,10 @@ zkSync Era supports the standard [Ethereum JSON-RPC API](https://ethereum.org/en
 
 ## RPC endpoint URLs
 
+:::warning Rate Limits
+
+We apply rate limiting to both HTTPs and Websocket APIs. The limits are generally permissive (currently 10s to 100s RPS per client), but please contact us if you face any issues in that regard.
+
```

Comment:
You need to end this `warning` section with a `  :::` else it will not close the section (see screenshot below)

<img width="800" alt="image" src="https://github.com/matter-labs/zksync-web-era-docs/assets/1890113/517ed459-ebcb-4d07-89e6-91c81620a0a8">